daemon/logger/syslog/syslog.go

```diff
@@ -207,3 +231,17 @@ func parseTLSConfig(cfg map[string]string) (*tls.Config, error) {
 
 	return tlsconfig.Client(opts)
 }
+
+func parseLogFormat(logFormat string) (syslog.Formatter, syslog.Framer, error) {
+	switch logFormat {
+	case "":
+		return syslog.UnixFormatter, syslog.DefaultFramer, nil
```

Comment:
Hi, current format keeps srslog behaviour as before the changes, please checkout the srs old code:
https://github.com/RackSec/srslog/blob/1155efef850aadc39aa84576f83a595a6fbdce74/srslog_unix.go#L42
formatter:  "<%d>%s %s[%d]: %s"
timestamp := time.Now().Format(time.Stamp)

Now for the master srslog code:
https://github.com/RackSec/srslog/blob/master/formatter.go
default:                "<%d> %s %s %s[%d]: %s"
timestamp := time.Now().Format(time.RFC3339)
unix				    "<%d>%s %s[%d]: %s"
timestamp := time.Now().Format(time.Stamp)
rfc3164					"<%d> %s %s %s[%d]: %s"
timestamp := time.Now().Format(time.Stamp)

So current code which defaults to unixformatter actually preserves original docker behaviour.